### PR TITLE
readme: improve development commands

### PR DESCRIPTION
**Pull request by dhimmel**

### CODE REVIEW

**Comment by dhimmel** [comment]
@epogrebnyak how does this look to you?

I thought about including some specific documentation of the pytest marks, but I actually think it may just be better to have the failure messages include those commands, since I don't like adding stuff to readme that easily gets outdated when we forget to update it with the code.

**Comment by epogrebnyak** [comment]
The failure messages of specific tests, in docstring? Good idea, it will be more up to date than readme.

**Comment by dhimmel** [comment]
> The failure messages of specific tests, in docstring? Good idea, it will be more up to date than readme.

Like you have at:

https://github.com/manubot/manubot/blob/0b7c14a6bbab77d6c91f0019dff524bb43587769/manubot/cite/tests/test_cite_command.py#L79-L82

Pytrest prints the docstrings for failed tests. I'm not sure whether the `Base_cite_command_render_stdout` docstring will display for failed tests, but the idea is that if so, it's the best place to suggest using `-m "not pandoc_version_sensitive"`.

File: README.md [hunk 1]
```diff
@@ -226,32 +226,42 @@ optional arguments:
...
+### Commands
+
+Below are some common commands used for development.
+They assume the working directory is set to the repository's root, and the conda environment is activated. 
 
 ```shell
+# run the test suite
+pytest
+
+# reformat python files for according to the black style rules (required to pass CI)
```
Comment (dhimmel) [inline README.md]:
```suggestion
# reformat Python files according to the black style rules (required to pass CI)
```

**Pull request merged**